Directory.Build.props

```diff
@@ -9,7 +9,7 @@
 
   <PropertyGroup>
     <Description>A client library for KMD Logic Audit</Description>
-    <VersionPrefix>1.3.1</VersionPrefix>
+    <VersionPrefix>1.3.2</VersionPrefix>
```

Comment:
As this is a minor release and not a patch, should we version it to 1.4.0?

Reply:
> As this is a minor release and not a patch, should we version it to 1.4.0?

yes, makes sense to me. Let me update to 1.4.0